src/connect.ts

```diff
@@ -75,7 +75,7 @@ async function connectWithFailFast(uri: string, client: MongoClient, logger: Con
     client.removeListener('serverHeartbeatFailed', heartbeatFailureListener);
     client.removeListener('serverHeartbeatSucceeded', heartbeatSucceededListener);
     logger.emit('devtools-connect:connect-attempt-finished', {
-      csfleVersionInfo: (client?.autoEncrypter as any /* NODE-4085 */)?.csfleVersionInfo
+      cryptSharedLibVersionInfo: (client?.autoEncrypter as any /* NODE-4085 */)?.cryptSharedLibVersionInfo
```

Comment:
The mentioned https://jira.mongodb.org/browse/NODE-4085 is closed - anything to do on that comment? Looks like the autoEncrypter is `js` though so we'll be keeping this `any` for the foreseeable future?

Reply:
> The mentioned https://jira.mongodb.org/browse/NODE-4085 is closed - anything to do on that comment? 

Yeah, bump it by about 200 tickets :stuck_out_tongue: NODE-4285 is the one for the renaming, we’ll need that until the next driver release.

> Looks like the autoEncrypter is `js` though so we'll be keeping this `any` for the foreseeable future?

The typings here come from the driver (which has separate typings for `AutoeEncrypter`), so this should go away at some point:)